src/gurt/telemetry.c

```diff
@@ -1295,6 +1329,88 @@ d_tm_print_stats(FILE *stream, struct d_tm_stats_t *stats, int format)
 	fprintf(stream, ", samples: %lu]", stats->sample_size);
 }
 
+static int
+_d_tm_reset_node(struct d_tm_context *ctx, struct d_tm_node_t *node)
```

Comment:
naming suggestion - I think we only want the d_tm_* prefix for public functions. Could just call this one "reset_node" if you like.

Reply:
ah, sure